fbw-a380x/src/systems/instruments/src/MFD/pages/ATCCOM/DAtisBlock.tsx

```diff
@@ -9,24 +9,33 @@ import { NewAtisIcon } from 'instruments/src/MFD/pages/ATCCOM/Elements/NewAtisIc
 import { AutoUpdateIcon } from 'instruments/src/MFD/pages/ATCCOM/Elements/AutoUpdateIcon';
 import { AutoPrintIcon } from 'instruments/src/MFD/pages/ATCCOM/Elements/AutoPrintIcon';
 
-interface DAtisBlockProps extends AbstractMfdPageProps {}
+interface DAtisBlockProps extends AbstractMfdPageProps {
+  readonly index: 1 | 2 | 3;
+  atisIcao: Subject<string | null> | Subscribable<string | null>;
```

Comment:
```suggestion
  atisIcao: Subscribable<string | null>;
```
Subject should basically never appear in interfaces. You want Accessible<T> if you just want .get(), Subscibable<T> if you want to subscribe, or MutableSubscribable<T> if you also need to be able to set it.

Reply:
Thanks for the note. I've removed all uses of `Subject<T>` in the interfaces and replaced them with `Subscribable<T>` instead.